doc/concepts/decor.rst

```diff
@@ -107,16 +107,20 @@ specialised on specific regions.
     .. code-block:: Python
 
         import arbor
+        from arbor import units as U
 
         # Create an empty decor.
         decor = arbor.decor()
 
         # Set cell-wide properties that will be applied by default to the entire cell.
-        decor.set_properties(Vm=-70, cm=0.02, rL=30, tempK=30+273.5)
+        decor.set_properties(Vm=-70 * U.mV,
+                             cm=0.02 * U.F / U.m2,
+                             rL=30 * U.Ohm * U.cm,
+                             tempK=30 * U.celsius) # or 303.15 K
```

Comment:
This isn't a criticism of the patch but seeing absolute temperature expressed as a multiple of a temperature in Celsius gives me the heebie-jeebies.

Reply:
Well, consider `tempK = 5 * U.micro * U.celsius`. 
Jokes aside, while overloading `*` to mean 'add unit X' is unsatisfying, it's how most units compose. And `celsius` being the main, useful exception, since it's used in the field _everywhere_.